sdk/sample/kfserving_sdk_sample.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Might be worth talking to @cliveseldon in the future to add a link to the SeldonDeploy API at the end of this notebook.